### PR TITLE
Fix INCLUDE handling when pathname is on a subsequent line

### PR DESCRIPTION
**Pull request by JeffreyALaw**
This fixes a reported bug in the handling of INCLUDE directives.

This may be controversial upstream as it does not handle multi-line pathnames.  Handling multi-line pathnames would require a line pushback mechanism or something similar AFAICT.  This has not been extensively tested.

### CODE REVIEW

File: gcc/fortran/scanner.c [hunk 1]
```diff
@@ -2115,6 +2115,38 @@ preprocessor_line (gfc_char_t *c)
...
+  b->file = current_file;
+  b->truncated = trunc;
+  wide_strcpy (b->line, line);
+
+  if (line_head == NULL)
+    line_head = b;
+  else
+    line_tail->next = b;
+
+      line_tail = b;
```
Comment (franred) [inline gcc/fortran/scanner.c]: I think this line has the wrong indentation.

Comment (JeffreyALaw) [reply gcc/fortran/scanner.c]: You're absolutely right.  I'm not sure how it was missed.

Comment (JeffreyALaw) [reply gcc/fortran/scanner.c]: I've updated my branch, which appears to automagically update the pull request.

**Comment by franred** [comment]
I've fixup both commits and merge this MR in: https://github.com/CodethinkLabs/gcc/pull/6

**Pull request closed without merging**